brokers/vscode/broker.go

```diff
@@ -94,10 +94,19 @@ func (b *Broker) PushEvents(tun *jsonrpc.Tunnel) {
 
 func (b *Broker) processPlugin(meta model.PluginMeta) error {
 	b.PrintDebug("Stared processing plugin '%s:%s'", meta.ID, meta.Version)
-	if meta.Attributes == nil || meta.Attributes["extension"] == "" {
-		return fmt.Errorf("VS Code extension field 'extension' is missing in description of plugin %s:%s", meta.ID, meta.Version)
+	if meta.Attributes == nil {
+		return fmt.Errorf("'attributes' are not found in VS Code extension description of the plugin %s:%s", meta.ID, meta.Version)
 	}
-	url := meta.Attributes["extension"]
+
+	url := meta.URL
+	extension := meta.Attributes["extension"]
+
+	if url == "" && extension == "" {
+		return fmt.Errorf("Neither 'extension' no 'url' attributes found in VS Code extension description of the plugin %s:%s", meta.ID, meta.Version)
+	} else if url != "" && extension != "" {
+		return fmt.Errorf("Both 'extension' and 'url' attributes found in VS Code extension description of the plugin %s:%s", meta.ID, meta.Version)
```

Comment:
It might not be absolutely clear what is the problem with this config. Could you change the error message to show that it is not allowed?